activerecord/lib/active_record/sanitization.rb

```diff
@@ -143,8 +143,9 @@ def disallow_raw_sql!(args, permit: connection.column_name_matcher) # :nodoc:
 
         if unexpected
           raise(ActiveRecord::UnknownAttributeReference,
-            "Query method called with non-attribute argument(s): " +
-            unexpected.map(&:inspect).join(", ")
+            "Query method called with non-attribute argument(s): " \
+            "#{unexpected.map(&:inspect).join(", ")}" \
+            "Known-safe values can be passed by wrapping them in Arel.sql()."
```

Comment:
Could you add a test?

Reply:
I think made test for the error information is redundant.

Reply:
Is there already an existing test?

Reply:
@zzak 
Sorry, I am sorry to reply so late.  
I checked this method is not tested.  
I added two simple tests.  
If you have time, please help again review, looking forward to your reply!